exporter/logstashexporter/sync.go

```diff
@@ -205,15 +205,21 @@ func (c *syncClient) publishWindowed(events []plog.LogRecord) (int, error) {
 
 func (c *syncClient) sendEvents(events []plog.LogRecord) (int, error) {
 	window := make([]interface{}, len(events))
+	//TODO: needs more tests
 	for i := range events {
-		record := events[i]
-		//TODO: Testing purpose, it should deserialize from LogRecord.Body, send it as-it-is?
-		event := beat.Event{
-			Timestamp: record.Timestamp().AsTime(),
-			Meta:      map[string]interface{}{},
-			Fields:    record.Attributes().AsRaw(),
+		logRecord := events[i]
+		logRecordBody, ok := newLogRecordBody(&logRecord)
+		// TODO: Possibly not a beats event, how should we handle it?
```

Comment:
we can drop it and log saying lumberjack exporter only handle events from beat

Reply:
following the [guidelines](https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md?plain=1#L363-L368) we should produce a metric: 

> Use logging carefully for events that can happen frequently to avoid flooding
the logs. Avoid outputting logs per a received or processed data item since this can
amount to a very large number of log entries (Collector is designed to process
many thousands of spans and metrics per second). For such high-frequency events
instead of logging consider adding an internal metric and incrementing it when
the event happens.